docker/Dockerfile.agent

```diff
@@ -0,0 +1,14 @@
+# use golang image to copy ssl certs later
+FROM golang:1.16
+
+FROM scratch
+ENV GODEBUG=netdns=go
+EXPOSE 3000
+HEALTHCHECK CMD ["/bin/woodpecker-agent", "ping"]
+
+# copy certs from golang:1.16 image
+COPY --from=0 /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/ca-certificates.crt
```

Comment:
```suggestion
# use alpine image to copy ssl certs later
FROM alpine:3.14.1
RUN apk add -U --no-cache ca-certificates

FROM scratch
ENV GODEBUG=netdns=go
EXPOSE 3000
HEALTHCHECK CMD ["/bin/woodpecker-agent", "ping"]

# copy certs from golang:1.16 image
COPY --from=0 /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/
```

Wouldn't it be more consistent to also use the alpine image here for the ca-certs?

Reply:
As the scratch image has nothing to do directly with the alpine image I would stick with getting it from the go image. This is additionally a quite common practice in many projects.

Reply:
fair enough